abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
> In other words, SetBytes() takes a variable-length integer of any size as input, and Bytes() produces a variable-length return

So you want to keep a reference to the original slice then in nist.Int ... ? For me it does not make sense to keep twice the information where it's not needed. If a user called `SetBytes` it can still keep the original reference to the slice, we should not need to duplicate the information (the original slice and the *squeezed* version)